crates/wac-resolver/src/lib.rs

```diff
@@ -1,19 +1,20 @@
 //! Modules for package resolvers.
 
+use crate::Error as WacResolutionError;
 use indexmap::IndexMap;
 use miette::{Diagnostic, SourceSpan};
-use wac_parser::Document;
+use thiserror::Error;
```

Comment:
nit: elsewhere we're using `thiserror::Error` qualified instead of through a `use`. Can you replace `Error` everywhere with `thiserror::Error`?